### PR TITLE
Object: Bunch

### PR DESCRIPTION
**Pull request by tylersticka**
## Overview

> A tight cluster of visual elements with a small amount of overlap. Useful for displaying avatars alongside bylines where multiple authors are equally credited.

## Screenshots

Screen Shot 2021-02-03 at 11 42 24 AM

## Testing

1. Confirm GitHub checks pass
2. Try out object in [deploy preview](https://deploy-preview-1130--cloudfour-patterns.netlify.app/?path=/docs/objects-bunch--of-avatars)

---

- See #487 #1093

### CODE REVIEW

File: src/objects/bunch/bunch.scss [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+ * 3. It feels odd to have the first element underneath the last. In lieu of a
+ *    a built-in CSS method of reversing the natural `z-index` order, we use
+ *    a custom property that we'll set with `nth-child` a little further down.
+ *
+ * @see https://github.com/w3c/csswg-drafts/issues/1026#issuecomment-385285148
+ */
+
+.o-bunch > * {
+  flex: none; /* 1 */
+  margin-right: (sizes.$overlap-small * -1); /* 2 */
```
Comment (Paul-Hebert) [inline src/objects/bunch/bunch.scss:31]: Smart! I likely would have ended up doing something like this instead (I like your version better)

```
.o-bunch > * + * {
   margin-left: (sizes.$overlap-small * -1); /* 2 */ 
}
```

File: src/objects/bunch/bunch.scss [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+}
+
+/**
+ * Set the `--bunch-index` property for the first ten elements. You can manually
+ * set `--bunch-index` for items past that point, but you should really question
+ * if this is the right CSS object to use if you need more than that!
+ */
+
+@for $i from 1 through 10 {
+  .o-bunch > *:nth-child(#{$i}) {
```
Comment (Paul-Hebert) [inline src/objects/bunch/bunch.scss:42]: Nit: is the `*` doing anything here?

**Review by Paul-Hebert — DISMISSED** [review]
This looks great Tyler!

I left one super minor code nit inline.

I've also got a design question. Are we concerned about this component breaking out of its container? Should we add `flex-wrap: wrap` to `.o-bunch` so that in the off chance there are too many items to fit in a container it breaks to the next line instead of breaking the layout?

File: src/objects/bunch/bunch.scss (hunk 2 above)
Comment (tylersticka) [reply src/objects/bunch/bunch.scss:42]: I don't think it is! Good catch

**Comment by tylersticka** [comment]
> I've also got a design question. Are we concerned about this component breaking out of its container? Should we add `flex-wrap: wrap` to `.o-bunch` so that in the off chance there are too many items to fit in a container it breaks to the next line instead of breaking the layout?

Good idea 👍

**Review by Paul-Hebert — APPROVED** [review]
Looks good!

**Pull request merged**